angular-material-calendar.js

```diff
@@ -264,10 +264,10 @@ angular.module("materialCalendar").directive("calendarMd", ["$compile", "$parse"
                     month: $scope.calendar.month + 1
                 };
                 setData();
-                handleCb($scope.onNextMonth, data);
+                handleCb($scope.onNextMonth, null, data);
             };
 
-            $scope.handleDayClick = function (date) {
+            $scope.handleDayClick = function ($event, date) {
```

Comment:
This breaks backwards compatibility, so it should be `$scope.handleDayClick = function(date, $event)` so as to be backwards compatible